src/main/java/com/javacode/entities/Comment.java

```diff
@@ -0,0 +1,87 @@
+package com.javacode.entities;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "comments")
+public class Comment {
+	@Id
+	@Column(name = "id")
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private int id;
+
+	@Column(name = "content")
+	private String content;
+
+	@Column(name = "reply_id")
+	private int reply_id;
+
+	@OneToOne(cascade = CascadeType.ALL)
+	@JoinColumn(name = "user_id", referencedColumnName = "id")
+	private User user;
+
+	@OneToOne(cascade = CascadeType.ALL)
```

Comment:
tương tự comment trên, xem lại quan hệ chỗ này nào

Reply:
Dạ vâng ạ

Reply:
Sao chưa fix giá phải là `many to one` chứ